tests/loan.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { expect, test } from '@playwright/test'
+import { LoanDto } from './dto/loan-dto'
+import { StatusCodes } from 'http-status-codes'
+
+test.describe('Loan tests', () => {
+  test('POST loan: returns 200 status code with law risk', async ({ request }) => {
+    const riskDto = LoanDto.calcPositiveRiskScore()
+    riskDto.loanPeriod = 12
+    const response = await request.post(
+      'https://backend.tallinn-learning.ee/api/loan-calc/decision',
+      {
+        data: riskDto,
+      },
+    )
+    expect.soft(response.status()).toBe(StatusCodes.OK)
+    const responseBody = await response.json()
+    expect.soft(responseBody.riskLevel).toBe('Low Risk')
+  })
+
+  test('POST loan: returns 200 status code with medium risk', async ({ request }) => {
+    const riskDto = LoanDto.calcPositiveRiskScore()
+    riskDto.loanPeriod = 9
+    const response = await request.post(
+      'https://backend.tallinn-learning.ee/api/loan-calc/decision',
+      {
+        data: riskDto,
```

Comment:
Just a tip. You could use the spread syntax https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

and rewrite line 26 as data: {...riskDto, loanPeriod: 9}
and then delete line 22 and you've saved yourself a line :-)

Reply:
Fixed